pkg/api/pod/warnings.go

```diff
@@ -252,6 +252,18 @@ func warningsForPodSpecAndMeta(fieldPath *field.Path, podSpec *api.PodSpec, meta
 				}
 			}
 		}
+		// warning on duplicate containers[*].ports
+		// in future release, once this is a validation error, the warning logic can be added only for update
```

Comment:
It will probably not be a validation error :(

Reply:
Updated.